wordpress-mobile/WordPressAuthenticator-iOS
# Redact tokens

Fixes #277 

WordPressAuthenticator is reliant on WordPressKit. Update the WordPressKit pod to the latest changes so that tokens are redacted in the console and activity log.

### To test

1. Check out this branch
2. `bundle exec pod install` (no errors should appear)
3. Build the project
4. Check out the host app PR to test these changes.

WPiOS: https://github.com/wordpress-mobile/WordPress-iOS/pull/14148
WCiOS: tk

**Note:** the podspec won't validate until WordPressKit `4.9.0-beta.1` has been published.

#### Do not merge until
1. The WordPressKit PR has been merged. Ref: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/245
2. A new WordPressKit release has been created
3. The release is published on Cocoapods trunk
4. This PR points to the new release and the podspec is updated
